apps/openchallenges/edam-etl/src/main.py

```diff
@@ -1,16 +1,75 @@
 """Extract, transform and load EDAM concepts"""
 
+import pandas as pd
+import requests
 from os import getenv
+from typing import Optional
+from typing import NoReturn
```

Comment:
nit: can be joined into a single statement:

```python
from typing import Optional, NoReturn
```

Reply:
Actually, maybe remove `NoReturn`?  Since it's not being used.

Reply:
thanks I forgot I had it there then didn't need it.